my.css

```diff
@@ -352,3 +356,770 @@ body {
+    max-height: 900px;
+    max-width: 1150px;
+  }
+
+  .bags-img {
+    display: block;
+    max-width: 45%;
+    height: 240px;
+    padding-left: 5px;
+  }
+
+  .bags-img1 {
+    z-index: 1;
+    width: 100%;
+    height: 540px;
+    padding-left: 0;
+  }
+}
+
+@media screen and (max-width: 768px) {
+  .bagscluth {
+    width: 100%;
+    max-height: unset;
+    margin: 0 auto;
+  }
+
+  .bagscluth__content {
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .bagclach__iner {
+    width: 100%;
+    padding-bottom: 50px;
+  }
+
+  .bagscluth__img {
+    max-width: 100%;
+    width: 100%;
+    height: auto;
+    justify-content: center;
+  }
+
+  .bags-img1 {
+    width: 250px;
+    height: 356px;
+    margin: 0;
+    padding: 0;
+  }
+
+  .bagscluth__footer {
+    display: none;
+  }
+
+  .bags-img {
+    display: none;
+  }
+
+  .card {
+    width: 320px;
+    padding: 10px 10px 15px 17px;
+    margin: 0 auto;
+    display: flex;
+    flex-flow: column;
+    background-color: #fff;
+    border: 1px solid #fff;
+    box-shadow:
+      29px 30px 22px 0px
+    rgba(0, 0, 0, 0.07);
+    box-sizing: border-box;
+  }
+
+  .card__header_font {
+    font-size: 20px;
+  }
+
+  .card__subheading {
+    margin: 0;
+    padding: 0;
+    font-size: 10px;
+    line-height: 10px;
+    letter-spacing: 0.07em;
+  }
+
+  .text {
+    font-size: 9px;
+    line-height: 13px;
+  }
+
+  .card__price-font {
+    font-size: 20px;
+    line-height: 28px;
+  }
+
+  .card__buttons {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    width: 140px;
+    height: 29px;
+  }
+
+  .card__choice {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    padding-top: 15px;
+  }
+
+  .card__choice-buttons {
+    display: none;
+  }
+
+  .card__row {
+    width: 275px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding-bottom: 5px;
+  }
+
+  .card__button {
+    width: 25px;
+    height: 25px;
+    border-radius: 100%;
+    border: unset;
+  }
+
+  .card__basket-button {
+    width: 100%;
+    height: 37px;
+    border: 1px solid #000;
+    box-sizing: border-box;
+    background-color: #fff;
+  }
+}
+
+/* Третья секция */
+
+.sectionlock {
+  margin: 0;
+  padding: 0;
+  position: relative;
+  min-height: 900px;
+  background:
+    linear-gradient(
+      90deg,
+      rgba(235, 227, 216, 1) 11%,
+      rgba(235, 227, 216, 1) 16%,
+      rgba(235, 227, 216, 1) 22%,
+      rgba(0, 0, 0, 1) 22%,
+      rgba(0, 0, 0, 1) 26%,
+      rgba(235, 227, 216, 1) 26%
+    );
+}
+
+.sectionlock__content {
+  display: flex;
+  max-width: 1150px;
+  max-height: 900px;
+  width: 100%;
+  margin: 0 auto;
+}
+
+.sectionlock__img {
+  width: 60%;
+  height: 844px;
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+}
+
+.sectionlock__img-clothes {
+  width: 780px;
+  height: 600px;
+  box-shadow:
+    0px 5px 18px 0px
+    rgba(0, 0, 0, 0.43
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline after "," in a multi-line function (function-comma-newline-after)](https://app.codacy.com/gh/WebHeroSchool/itog/pullRequest?prid=5642160)